### DISCUSSION TITLE
pt/quickstart/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#9704] [Docs]
# pt/quickstart/

Explora vários métodos para instalar Ultralytics usando pip, conda, git e Docker. Aprende a utilizar Ultralytics com a interface de linha de comandos ou nos teus projectos Python .

https://docs.ultralytics.com/pt/quickstart/

**Comment by jonatapaulino** [comment]
Oops! I always follow your videos on YouTube. Thanks for teaching agent. I have a question: Is there any way to refine the hyperparameters in yolov8? I have a project that has an accuracy of 67, but I need to improve this performance a lot. Can you help me? Thanks.

**Reply by glenn-jocher** [reply]
Hey, thanks for the kind words and for following our videos! 🙌 To refine the hyperparameters for YOLOv8 and improve your model's accuracy, you can experiment with various settings in your training configuration file (usually a `.yaml` file). Here's a quick example of how you might adjust some common hyperparameters:

```yaml
# Example of hyperparameters in a YAML configuration file
lr0: 0.01  # initial learning rate
lrf: 0.1   # final learning rate factor
momentum: 0.937  # optimizer momentum
weight_decay: 0.0005  # weight decay
warmup_epochs: 3  # number of warmup epochs
warmup_momentum: 0.8  # initial momentum during warmup
warmup_bias_lr: 0.1  # initial learning rate for biases during warmup
```

You can modify these values to see if they help improve performance. Additionally, consider using techniques like data augmentation, increasing the dataset size, or even using a more complex model variant if your hardware allows. For a detailed guide on tuning these parameters, check out the [Ultralytics documentation](https://docs.ultralytics.com/). Good luck with your project! 🚀

**Reply by jonatapaulino** [reply]
Thanks for responding, I'm glad there's a follow-up with readers. One question: When you say: "even use a more complex model variant if your hardware allows it". What variants could these be? Thanks again.

**Reply by glenn-jocher** [reply]
@jonatapaulino absolutely! For more complex model variants, you can explore YOLOv8's larger models like YOLOv8m (medium), YOLOv8l (large), and YOLOv8x (extra-large). These models have more parameters and can potentially improve accuracy if your hardware can handle the increased computational load. Here's a quick example of how to switch to a larger model:

```bash
# Train with a larger model variant
yolo train data=coco128.yaml model=yolov8l.pt epochs=100 imgsz=640
```

Feel free to experiment with these variants and see which one works best for your project! 😊 For more details, check out the [Ultralytics documentation](https://docs.ultralytics.com/quickstart/).

**Reply by jonatapaulino** [reply]
@glenn-jocher my project is a serum drop counter and it will have to run on a cell phone. The nurse points her cell phone at the serum drop counter and specifies how many drops she wants to fall. This counter has to tell her the amount of drops as quickly as possible, and I specified that it tell her this after 3 drops. I even managed to do it, but it is making an error in the count precisely due to the lack of precision. I don't know if I was clear, I hope you understood.

**Reply by glenn-jocher** [reply]
Hi there! For your serum drop counter project on a cell phone, precision is key. Consider using a more robust model like YOLOv8m or YOLOv8l for better accuracy. Also, ensure your model is well-optimized for mobile deployment. You can use the `export` function to convert your model to a format like TensorFlow Lite for efficient mobile inference:

```python
from ultralytics import YOLO

# Load and export the model
model = YOLO('yolov8n.pt')
model.export(format='tflite')
```

This should help improve the precision and performance of your drop counter. Good luck! 😊
